### PR TITLE
Feature/scheduled event tag filter

### PR DESCRIPTION
**Pull request by yevhen-turovets**
The task in Trello https://trello.com/c/Hh2LsLnl

### CODE REVIEW

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            <VCard>
+                <VList>
+                    <VListItem>
+                        <VListItemContent>
+                            <VTextField
+                                solo
+                                hide-details
+                                dense
+                                flat
+                                color="#2C2C2C"
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:41]: hard coded color constant

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                                background-color="rgba(224, 224, 224, 0.3)"
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:42]: hard coded color constant

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+                return isNaN(number) ? item : number;
+            });
+        }
+    }
+};
+</script>
+
+<style scoped>
+.no-items {
+    color: #2c2c2c;
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:285]: hard coded color

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+            eventTypesTagsChecked: [],
+            endDate: new Date().toLocaleDateString()
+        };
+    },
+
+    watch: {
+        $route: 'setEventTypesTagsFilter'
+    },
+
+    async mounted() {
```
Comment (mv-v-7) [inline client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:158]: is it really necessary

**Review by mv-v-7 — CHANGES_REQUESTED** [review]

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue (hunk 1 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:41]: The color constant is not working their

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue (hunk 2 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:42]: The color constant is not working their

File: client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue (hunk 4 above)
Comment (yevhen-turovets) [reply client/src/components/scheduled-events/Past/Filter/EventTypesTags.vue:158]: Why not?)

**Review by mv-v-7 — APPROVED** [review]
At a whole looks good.  async mounted are discussible

**Review by aLLeXUs — APPROVED** [review]

**Pull request merged**